test/units/srcset_test.rb

```diff
@@ -760,4 +760,29 @@ def srcset
       )
     end
   end
+
+  class SrcsetDisablePathEncoding < Imgix::Test
+    include SrcsetTest
+
+    def test_enable_path_encoding
```

Comment:
Nit, feel free to ignore. Path encoding isn't "enabled" but rather "default" behavior.

```suggestion
    def test_encode_path_by_default
```